### PR TITLE
Remove some more variable variables + some test cleanup 

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Remove some more variable variables

Before
----------------------------------------
```
$$dv  = NULL
```

After
----------------------------------------
```$endDate = NULL```

Technical Details
----------------------------------------
Ug

Comments
----------------------------------------
Ug

### CODE REVIEW

File: CRM/Member/Form/Membership.php [hunk 1]
```diff
@@ -1379,17 +1379,12 @@ public function submit() {
...
-          $$dv = NULL;
-          foreach ($this->_memTypeSelected as $memType) {
-            $membershipTypeValues[$memType][$dv] = NULL;
-          }
+        foreach ($this->_memTypeSelected as $memType) {
+          $membershipTypeValues[$memType]['joinDate'] = NULL;
+          $membershipTypeValues[$memType]['startDate'] = NULL;
+          $membershipTypeValues[$memType]['endDate'] = NULL;
         }
+        $endDate = $startDate = NULL;
```
Comment (seamuslee001) [inline CRM/Member/Form/Membership.php:1387]: why no $joinDate?

Comment (eileenmcnaughton) [reply CRM/Member/Form/Membership.php:1387]: because the ide suggested it wasn't used

Comment (seamuslee001) [reply CRM/Member/Form/Membership.php:1387]: ok

**Comment by seamuslee001** [comment]
Looks good to me merging

**Pull request merged**